src/hooks/index.ts

```diff
@@ -0,0 +1,7 @@
+import useCountdown from './useCountdown'
+import usePrevious from './usePrevious'
+
+export default {
+  useCountdown,
+  usePrevious,
+}
```

Comment:
In the `index.ts` files, I wanted use the `export { default as useCountdown } from './useCountdown'` syntax but I didn't manage to make it work for some reason 🤷‍♂️